observations/obs_converters/Ameriflux/fluxnetfull_to_obs.f90

```diff
@@ -0,0 +1,1180 @@
+
+  tower%startstring    = 'TIMESTAMP'
+  tower%endstring      = 'TIMESTAMP'
+  res                  = .false.  
+
+  write(string1, *) 'Time resolution is set to =', time_resolution
+  write(string2, *) 'Using TIMESTAMP to set DART observation time'
+  if (verbose) call error_handler(E_MSG,'Fluxnetfull_to_obs',string1,text2=string2)
+
+elseif (time_resolution == 'HH' .or. time_resolution == 'HR' .or. &
+        time_resolution == 'WW') then
+
+  res = .true.
+  write(string1, *) 'Time resolution is set to =', time_resolution
+  write(string2, *) 'Using TIMESTAMP_START and TIMESTAMP_END to set: DART observation time'
+  if (verbose) call error_handler(E_MSG,'Fluxnetfull_to_obs',string1,text2=string2)
+
+else
+  write(string1,*) 'time_resolution set incorrectly within input.nml'
+  write(string2,*) 'time_resolution must be HR,HH,DD,WW,MM, or YY'
+  call error_handler(E_ERR, source, string1,text2=string2)
+
+endif
+
+! 2) energy_balance: .true. changes sensible and latent heat strings
+!    Note: There are no qc values for energy_balance correction
+!    The qc values are manually set later in code 
+if (energy_balance .eqv. .true.) then
+
+   tower%lestring       = 'LE_CORR'
+   tower%leUNCstring    = 'LE_CORR_JOINTUNC'
+   tower%hstring        = 'H_CORR'
+   tower%hUNCstring     = 'H_CORR_JOINTUNC'
+   write(string1,*) 'WARNING! Energy balance correction data turned on for LE and H'
+   write(string2,*) 'Check to make sure LE_CORR and H_CORR data is not missing'
+   call error_handler(E_MSG, source, string1,text2=string2)
+else
+
+   write(string1,*) 'Standard LE (LE_F_MDS) and H (H_F_MDS) data being used'
+   call error_handler(E_MSG, source, string1)
+endif
+
+! Specify the maximum number of observations in the input file,
+! but only the actual number created will be written out.
+! Each line has 5 flux observations available.
+! Each observation in this series will have a single
+! observation value and a quality control flag.  
+! Initialize two empty observations - one to track location
+! in observation sequence - the other is for the new observation.
+
+iunit = open_file(text_input_file, 'formatted', 'read')
+if (verbose) call error_handler(E_MSG,'Fluxnetfull_to_obs','opened input file '//trim(text_input_file))
+
+nlines     = count_file_lines(iunit)
+max_obs    = 5*nlines
+num_copies = 1
+num_qc     = 1
+first_obs  = .true.
+
+call static_init_obs_sequence()
+call init_obs(obs,      num_copies, num_qc)
+call init_obs(prev_obs, num_copies, num_qc)
+call init_obs_sequence(obs_seq, num_copies, num_qc, max_obs)
+
+! the first one needs to contain the string 'observation' and the
+! second needs the string 'QC'.
+call set_copy_meta_data(obs_seq, 1, 'observation')
+call set_qc_meta_data(  obs_seq, 1, 'Fluxnet QC')
+
+! Subroutine decode_header reads obs file header and identifies the columns 
+! where flux variables of interest are located
+rewind(iunit)
+call decode_header(iunit, nwords)
+
+obsloop: do iline = 2,nlines
```

Comment:
Could you please point me to the obs_seq.out that you generated in the test?

Reply:
Sure --  I put them here:  /glade/scratch/bmraczka/Ameriflux_data/